aten/src/ATen/native/quantized/cpu/qnnpack/src/fc-prepack.cc

```diff
@@ -0,0 +1,53 @@
+#include <pytorch_qnnpack.h>
+#include <qnnpack/pack.h>
+#include <qnnpack_func.h>
+#include <cstring>
+
+namespace qnnpack {
+PackBMatrix::PackBMatrix(
+    size_t input_channels,
+    size_t output_channels,
+    uint8_t kernel_zero_point,
+    float kernel_scale,
+    const uint8_t* kernel,
+    const int32_t* bias) {
```

Comment:
Same comment regarding const correctness applies here and the rest of the commit.  I won't repeat myself. :)